pages/unit/_unit.vue

```diff
@@ -23,14 +23,14 @@ export default {
     },
     methods: {
         getUnit() {
+            let activeIndex = this.$store.getters['currentSeason'];
```

Comment:
Codacy has a fix for the issue: [Replace `'currentSeason'` with `"currentSeason"`](https://app.codacy.com/gh/Eight-Corner/front_toche_web/pullRequest?prid=9689045)
```suggestion
            let activeIndex = this.$store.getters["currentSeason"];
```